<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>1063</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="iconOnlyWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="logo">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../Resource.qrc">:/Icon/icon/simulation.ico</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_settingsPage">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/settings-5-32.ico</normaloff>
             <normalon>:/Icon/icon/settings-5-32 blauw.ico</normalon>:/Icon/icon/settings-5-32.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_IOPage">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/settings-12-32.ico</normaloff>
             <normalon>:/Icon/icon/settings-12-32 blauw.ico</normalon>:/Icon/icon/settings-12-32.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simPage">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/play-6-32.ico</normaloff>
             <normalon>:/Icon/icon/play-6-32 blauw.ico</normalon>:/Icon/icon/play-6-32.ico</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>890</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Exit">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Icon/icon/close-window-64.ico</normaloff>
           <normalon>:/Icon/icon/close-window-64 blauw.ico</normalon>:/Icon/icon/close-window-64.ico</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="fullMenuWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="logo2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>simulation.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="logoText">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PID regelaar sim</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_settingsPage2">
           <property name="text">
            <string>Simulatie instellingen</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/settings-5-32.ico</normaloff>
             <normalon>:/Icon/icon/settings-5-32 blauw.ico</normalon>:/Icon/icon/settings-5-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_IOPage2">
           <property name="text">
            <string>I/O instellingen</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/settings-12-32.ico</normaloff>
             <normalon>:/Icon/icon/settings-12-32 blauw.ico</normalon>:/Icon/icon/settings-12-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_simPage2">
           <property name="text">
            <string>Simulatie</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Icon/icon/play-6-32.ico</normaloff>
             <normalon>:/Icon/icon/play-6-32 blauw.ico</normalon>:/Icon/icon/play-6-32.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>838</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit2">
         <property name="text">
          <string>EXIT</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Icon/icon/close-window-64.ico</normaloff>
           <normalon>:/Icon/icon/close-window-64 blauw.ico</normalon>:/Icon/icon/close-window-64.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>14</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resource.qrc">
              <normaloff>:/Icon/icon/menu-4-32.ico</normaloff>:/Icon/icon/menu-4-32.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>599</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="IPAdress">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>192.168.1.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../Resource.qrc">
                <normaloff>:/Icon/icon/usb-disconnected-32.ico</normaloff>
                <normalon>:/Icon/icon/usb-connected-32.ico</normalon>:/Icon/icon/usb-disconnected-32.ico</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>14</width>
                <height>14</height>
               </size>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="settingsPage">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>460</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="IOPage"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_10</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconOnlyWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>toggled(bool)</signal>
   <receiver>fullMenuWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_settingsPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_settingsPage2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_IOPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_IOPage2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_simPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_simPage2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_settingsPage2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_settingsPage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_IOPage2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_IOPage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_simPage2</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_simPage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
